service-api/src/main/java/greencity/dto/openinghours/OpenHoursDto.java

```diff
@@ -0,0 +1,27 @@
+package greencity.dto.openinghours;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import greencity.dto.breaktime.BreakTimeDto;
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class OpenHoursDto implements Comparable<OpenHoursDto> {
```

Comment:
Is this class used somewhere?

Reply:
removed. it used in PlaceService. I will move it in my next task.